exercises/spiral-matrix/src/main/java/SpiralMatrixBuilder.java

```diff
@@ -0,0 +1,5 @@
+class SpiralMatrixBuilder {
+
+
+
+}
```

Comment:
I think there's an open issue for cleanup; let's decide there whether we want no implementation or empty class(es)! https://github.com/exercism/java/issues/554